src/ipmi.c

```diff
@@ -200,6 +211,19 @@ static void sensor_read_handler(ipmi_sensor_t *sensor, int err,
   plugin_dispatch_values(&vl);
 } /* void sensor_read_handler */
 
+static const char *sensor_get_db_type(ipmi_sensor_t *sensor) {
```

Comment:
As far I remember, use of only one `ipmi_sensor_get_base_unit()` is not enough for a such task.
The sensor, besides the base unit, also has a 'modifier' and 'rate' values (ipmi_sensor_get_modifier_unit() and ipmi_sensor_get_rate_unit() respectively). May be these are rare used, but I think we need to check their values anyway - otherwise incorrect result can be obtained.